app/models/discount_code.py

```diff
@@ -23,7 +23,11 @@ class DiscountCode(SoftDeletionModel):
     valid_till = db.Column(db.DateTime(timezone=True), nullable=True)
     event_id = db.Column(db.Integer, db.ForeignKey('events.id', ondelete='CASCADE'))
     event = db.relationship('Event', backref='discount_codes', foreign_keys=[event_id])
+<<<<<<< HEAD
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible SCM diff in code](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4778007)